meteor/sofie-server/business-logic/services/test/rundown-timeline-service.spec.ts

```diff
@@ -0,0 +1,136 @@
+import { anything, instance, mock, verify, when } from 'ts-mockito'
+import { Rundown, RundownInterface } from '../../../model/entities/rundown'
+import { RundownTimelineService } from '../timeline-integration/rundown-timeline-service'
+import { RundownEventEmitter } from '../interfaces/rundown-event-emitter'
+import { RundownRepository } from '../../../data-access/repositories/interfaces/rundown-repository'
+import { TimelineRepository } from '../../../data-access/repositories/interfaces/timeline-repository'
+import { AdLibPieceRepository } from '../../../data-access/repositories/interfaces/ad-lib-piece-repository'
+import { TimelineBuilder } from '../interfaces/timeline-builder'
+import { RundownEventBuilder } from '../interfaces/rundown-event-builder'
+import { RundownActiveException } from '../../../model/exceptions/rundown-active-exception'
+import { RundownEventType } from '../../../model/enums/rundown-event-type'
+
+describe(`${RundownTimelineService.name}`, () => {
+	describe(`${RundownTimelineService.prototype.deleteRundown.name}`, () => {
+		// eslint-disable-next-line jest/expect-expect
+		it('deletes a rundown, when it receives a valid RundownId', async () => {
+			const mockRundownRepository: RundownRepository = mock<RundownRepository>()
+
+			const randomRundownId: string = 'randomRundownId'
+			const randomRundown: Rundown = createInactiveRundown(randomRundownId)
+
+			when(mockRundownRepository.getRundown(randomRundownId)).thenReturn(Promise.resolve(randomRundown))
+
+			const testee: RundownTimelineService = createTestee({ rundownRepository: instance(mockRundownRepository) })
+
+			await testee.deleteRundown(randomRundownId)
+
+			verify(mockRundownRepository.deleteRundown(randomRundownId)).once()
+		})
+
+		// eslint-disable-next-line jest/expect-expect
+		it('builds a rundown deleted event, when it receives a valid RundownId', async () => {
+			const mockRundownRepository: RundownRepository = mock<RundownRepository>()
+			const mockRundownEventBuilder: RundownEventBuilder = mock<RundownEventBuilder>()
+
+			const randomRundownId: string = 'randomRundownId'
+			const randomRundown: Rundown = createInactiveRundown(randomRundownId)
+
+			when(mockRundownRepository.getRundown(randomRundownId)).thenReturn(Promise.resolve(randomRundown))
```

Comment:
Requested change applied.

Had issues early on with resolving a `Promise<boolean>`, which resultet in me using that syntax, as the worked for resolving that one. Since then where i had the `Promise<boolean>`, has been reworked to be `Promise<void>`